model/ios/registry.yaml

```diff
@@ -1,45 +1,43 @@
 groups:
-  - id: registry.ios.deprecated
+  - id: registry.ios
     type: attribute_group
-    display_name: Deprecated iOS Attributes
+    display_name: iOS Attributes
     brief: >
-        The iOS platform on which the iOS application is running.
+      The iOS platform on which the iOS application is running.
     attributes:
       - id: ios.state
-        stability: experimental
-        deprecated: "Moved to a payload field of `device.app.lifecycle`."
+        stability: development
+        brief: >
+          This attribute represents the state of the application.
         note: >
           The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate),
           and from which the `OS terminology` column values are derived.
```

Comment:
what are the `OS terminology` and `OS identifiers` things?

Reply:
I'm literally just moving things around and not trying to rewrite/redefine/clarify too much.